specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-11-01/computeRPCommon.json

```diff
@@ -1883,6 +1883,15 @@
         }
       },
       "description": "Specifies additional settings to be applied when patch mode AutomaticByPlatform is selected in Linux patch settings."
+    },
+    "ServiceArtifactReference": {
```

Comment:
any idea if this property will be used in other resources in Compute? 

If this is VMSS specific resource, it should be in virtualMachineScaleSet.json file, 
if there are plans to add this property to other resources, this is good here.

Reply:
It is specific to VMSS, will move it inside the virtualMachineScaleSet.json file